src/containers/enrollment/EnrollmentLink.js

```diff
@@ -0,0 +1,42 @@
+import { faCopy } from '@fortawesome/pro-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// @flow
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardSegment,
+} from 'lattice-ui-kit';
+
+import copyToClipboard from '../../utils/copyToClipboard';
+
+const EnrollmentLink = () => {
+  const enrollmentLinkQueryParams = window.location.hash.split('?')[1];
+  const enrollmentLink = `http://openlattice.com/login?${enrollmentLinkQueryParams}`;
+  return (
+    <>
+      <Card>
+        <CardHeader mode="danger">
+          Page Not Found
+        </CardHeader>
+        <CardSegment>
+          You have tried to open a mobile link in a web browser. Please install the Chronicle mobile application
+          and open the link on a mobile device.
```

Comment:
We need to update this message to apply to iOS users as well.

Reply:
ohh hmm i was thinking this message was generic enough to apply to either iOS or android? what do you think?

Reply:
ah never mind. Thought I saw "Android" in the text